### ISSUE TITLE
Write something about using Numbas with ClaroRead

### ISSUE THREAD

**Issue by christianp** [numbas/Numbas#987]

**Comment by christianp** [comment]
Scarlett looked at ClaroRead and the short story is that it doesn't read maths at all - it seems to just grab elements' text content, instead of looking at the accessibility tree.
